cloud-functions/functions/custom-logo.js

```diff
@@ -0,0 +1,50 @@
+// this is meant to be run by Covid Watch devs to set a custom logo for different organizations to appear in the NavBar
+// perhaps at some point this could be adapted for an admin user to set their organization's logo
+
+const admin = require('firebase-admin');
+const fs = require('fs');
+
+// these desktop paths for testing purposes, they work!  simulating a dev downloading a file from browser, saving to desktop, then referencing in this script to upload
+const soylentLogoPath = '../../../../soylent-green-logo.png'; // https://fanart.tv/fanart/movies/12101/hdmovielogo/soylent-green-51744f6296779.png
```

Comment:
Let's not have these values hardcoded in the final commit

Reply:
removed